### DISCUSSION TITLE
Save photo edits without needing to save it as a new photo

### DISCUSSION THREAD

**Discussion by ymilly** [ente-io/ente#4299] [Enhancements]
It would be nice to edit a photo but allowing the edits to "mask" the original photo while retaining the originals metadata (timestamp, album order etc.)

Currently, any edits require creating a photo copy and this photo is treated as a new photo (with a new timestamp, I believe.

**Comment by samsonjs** [comment]
Yeah I want non-destructive edits that are synced, like Apple's Photos apps.

**Comment by henrivdm** [comment]
The new photo after edit seems to lose all the original photo metadata, which does not work for me: at least give an option to edit the original, or to create a copy with the original photo metadata.
